<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompilerSettings</class>
 <widget class="QWidget" name="CompilerSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
   <item>
    <layout class="QVBoxLayout" name="leftLayout">
     <item>
      <widget class="QLabel" name="compilersListLabel">
       <property name="text">
        <string>Compilers</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="compilersList"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="importExportLayout">
       <item>
        <widget class="QPushButton" name="importButton">
         <property name="text">
          <string>Import</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exportButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Export</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="addRemoveLayout">
       <item>
        <widget class="QComboBox" name="addCompilerComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeCompilerButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="settingsLayout">
     <property name="title">
      <string>Compiler Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="settingTopLayout">
        <item>
         <layout class="QVBoxLayout" name="compilerTypeAndNameLayout">
          <item>
           <layout class="QHBoxLayout" name="compilerTypeLayout" stretch="0,0,0">
            <item>
             <widget class="QLabel" name="compilerTypeHeaderLabel">
              <property name="text">
               <string>Compiler Type:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="compilerTypeLabel">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="compilerNameLayout">
            <item>
             <widget class="QLabel" name="compilerNameLabel">
              <property name="text">
               <string>Name</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="compilerName">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="saveCompilerButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="compilerSettingsMainLayout" stretch="0,0,1">
        <item>
         <widget class="QTableWidget" name="compilerSpecificSettingsTable">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Value</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="environmentVariablesLabel">
          <property name="text">
           <string>Environment Variables</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="environmentVariablesLayout"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
